test/Common.test.cpp

```diff
@@ -22,10 +22,10 @@ TEST(String, split) {
 	EXPECT_EQ((NAS2D::StringList{"ab", "c"}), NAS2D::split("ab.c", '.'));
 	EXPECT_EQ((NAS2D::StringList{"abc"}), NAS2D::split("abc.", '.'));
 
-	EXPECT_EQ((NAS2D::StringList{"a", "b", "c"}), NAS2D::split("a,b,c", ',', false));
+	EXPECT_EQ((NAS2D::StringList{"a", "b", "c"}), NAS2D::split("a,b,c", ','));
 	EXPECT_EQ((NAS2D::StringList{"abc"}), NAS2D::split("abc", ',', false));
-	EXPECT_EQ((NAS2D::StringList{"", "abc"}), NAS2D::split(",abc", ',', false));
-	EXPECT_EQ((NAS2D::StringList{"a", "bc"}), NAS2D::split("a,bc", ',', false));
-	EXPECT_EQ((NAS2D::StringList{"ab", "c"}), NAS2D::split("ab,c", ',', false));
-	// EXPECT_EQ((NAS2D::StringList{"abc", ""}), NAS2D::split("abc,", ',', false));  // Actual: {"abc"}
+	EXPECT_EQ((NAS2D::StringList{"", "abc"}), NAS2D::split(",abc", ','));
+	EXPECT_EQ((NAS2D::StringList{"a", "bc"}), NAS2D::split("a,bc", ','));
+	EXPECT_EQ((NAS2D::StringList{"ab", "c"}), NAS2D::split("ab,c", ','));
+	// EXPECT_EQ((NAS2D::StringList{"abc", ""}), NAS2D::split("abc,", ','));  // Actual: {"abc"}
```

Comment:
Looks like the `false` parameter tests were updated correctly for the method splitting.

The default parameter tests should be updated to the `splitSkipEmpty` variant. As a side note, now that the methods have been split, we should probably split the unit tests into `split` and `splitSkipEmpty`.

We should probably swap the order of the unit tests so that `split` tests are done first, followed by `splitSkipEmpty`. This would match the updated order of the methods in the implementation file. Barring any reason not to, we should try to be consistent about order.

----

Currently I'm seeing some unit test failures. I suspect this is related to the default parameter tests not being updated to use `splitSkipEmpty`.

Reply:
Yeah I'm still getting "Can not find gtest.lib" on my machine so I can't test the test project locally, it keeps failing to build. :(

Reply:
Hmm, I thought that was resolved after the discussion in #184. The project switched to using `vcpkg` to install Google Test. Maybe reopen that Issue if it's still a problem.

Reply:
`vcpkg install gtest:x86/x64-windows` fixed it for the main project but not the test project. If I try to run it again it says it's already installed.

Reply:
Err, Google Test is only used by the test project, not by the main project. There's something more going on here. We may need to investigate further.